lib/kube/proxy/utils_test.go

```diff
@@ -184,6 +205,17 @@ func setupTestContext(ctx context.Context, t *testing.T, cfg testConfig) *testCo
 		OnReconcile:      cfg.onReconcile,
 	})
 	require.NoError(t, err)
+	// create session recording path
+	// testCtx.kubeServer.DataDir/log/upload/streaming/default
+	err = os.MkdirAll(
```

Comment:
What's going to clean up this directory? Should you use `t.TempDir` instead?

Reply:
`testCtx.kubeServer.DataDir` is already a [`t.TempDir`](https://github.com/gravitational/teleport/blob/a647ca4254d5f0db34f4819decf325d8ed47c188/lib/kube/proxy/utils_test.go#L172). I am just creating the correct folder structure for storing the sessions.

Reply:
Ah, got it!